packages/components/tests/dummy/app/templates/components/pagination-bar.hbs

```diff
@@ -65,33 +206,236 @@
 <section>
   <h3 class="dummy-h3" id="accessibility"><a href="#accessibility" class="dummy-link-section">§</a> Accessibility</h3>
 
-  {{! ADD YOUR CONTENT HERE }}
-
+  <!-- <Doc::WcagList
+    @criteriaList={{array
+      "1.3.1"
+      "1.3.2"
+      "1.4.1"
+      "1.4.3"
+      "1.4.4"
+      "1.4.10"
+      "1.4.11"
+      "1.4.12"
+      "2.1.1"
+      "2.1.2"
+      "2.4.3"
+      "2.4.4"
+      "2.4.6"
+      "2.4.7"
+      "2.5.3"
+      "3.2.1"
+      "4.1.2"
+    }}
+  /> -->
 </section>
 
 <section data-test-percy>
   <h3 class="dummy-h3" id="showcase">
     <a href="#showcase" class="dummy-link-section">§</a>
     Showcase
   </h3>
+  <button id="dummy-toggle-highlight" type="button" {{on "click" this.toggleHighlight}}>
+    {{if this.showHighlight "Hide" "Show"}}
+    layout highlight
+  </button>
+
+  <div class="{{if this.showHighlight 'dummy-pagination-layout-highlight'}}">
+
+    <h4 class="dummy-h4">Base elements</h4>
+
+    <h5 class="dummy-h5">TotalCount</h5>
+
+    <span class="dummy-text-small">with <code class="dummy-code">@totalItems</code></span>
+    <Hds::PaginationBar::TotalCount @itemsPerPage={{10}} @currentPage={{1}} @totalItems={{103}} />
+
+    <!--
+    <span class="dummy-text-small">without <code class="dummy-code">@totalItems</code></span>
+    <Hds::PaginationBar::TotalCount @itemsPerPage={{10}} @currentPage={{1}} />
+    -->
+
+    <span class="dummy-text-small">with
+      <code class="dummy-code">@totalItems</code>
+      but with
+      <code class="dummy-code">@hasTotalItems=false</code></span>
+    <Hds::PaginationBar::TotalCount
+      @itemsPerPage={{10}}
+      @currentPage={{1}}
+      @totalItems={{103}}
+      @hasTotalItems={{false}}
+    />
+
+    <!--
+    <span class="dummy-text-small">without
+      <code class="dummy-code">@totalItems</code>
+      but with
+      <code class="dummy-code">@hasTotalItems=true</code></span>
+    <Hds::PaginationBar::TotalCount @itemsPerPage={{10}} @currentPage={{1}} @hasTotalItems={{true}} />
+    -->
+
+    <h5 class="dummy-h5">PageSize</h5>
+    <Hds::PaginationBar::PageSize @sizes={{array 10 30 50}} />
+
+    <h5 class="dummy-h5">Pagination::NavButton</h5>
+    {{#let (array "compact" "normal") as |types|}}
+      {{#each types as |type|}}
```

Comment:
something in these examples is not working, because I don't see the "disabled" state of the prev/next elements

Reply:
I'm seeing the disabled states as expected. Are you positive you have been looking at the latest updates? (I pushed all the updates I made from your previous round of feedback Monday.)

<img width="1001" alt="Screen Shot 2022-12-06 at 10 04 14 AM" src="https://user-images.githubusercontent.com/108769823/205987796-aee14710-6c74-48cb-9f53-80c8b05761f4.png">